airflow/utils/db_cleanup.py

```diff
@@ -370,3 +383,19 @@ def run_cleanup(
                 session=session,
             )
             session.commit()
+
+
+@provide_session
+def export_archived_records(export_format, output_path, session: Session = NEW_SESSION):
+    """Export cleaned data to the given output path in the given format."""
+    inspector = inspect(session.bind)
+    table_names = inspector.get_table_names()
+    for table_name in table_names:
+        if table_name.startswith(ARCHIVE_TABLE_PREFIX):
+            _dump_db(
+                target_table=table_name,
+                file_path=os.path.join(output_path, f"{table_name}.{export_format}"),
+                export_format=export_format,
+                session=session,
+            )
```

Comment:
I feel like we should have a flag to drop the table as well?

Reply:
Also thinking of adding table names so one can export specific archive tables. WDYT?

Reply:
SGMT, sorta like we do on the clean command.